calico/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+calico-ctl:
```

Comment:
I would really like to see these combined into one command but understand there are technical reasons why it can't be. Can you make sure you raised appropriate issues to track those technical issues before this gets merged.